### ISSUE TITLE
Collator Switching

### ISSUE THREAD

**Issue by 2xJDubs** [darwinia-network/collator-staking-ui-v2#26]
Is it possible to be able to change collators without unstaking?  Of course this should only be possible after the 24hr staking lockout.

**Comment by Annawoody** [comment]
Yes, it is possible to change collators without unstaking, as long as you have completed the 24-hour staking lockout period. After this period, you should be able to switch collators without needing to withdraw your funds first. This allows you to keep your funds staked while you choose a new collator to delegate to. Make sure to follow the specific instructions provided by the platform you are using to ensure the process is done correctly.
@2xJDubs

**Comment by 2xJDubs** [comment]
> Yes, it is possible to change collators without unstaking, as long as you have completed the 24-hour staking lockout period. After this period, you should be able to switch collators without needing to withdraw your funds first. This allows you to keep your funds staked while you choose a new collator to delegate to. Make sure to follow the specific instructions provided by the platform you are using to ensure the process is done correctly. @2xJDubs

Not sure who you are or what your angle is here but I can guarantee that it is not possible to switch collators without unstaking first.  I know my way around our staking dashboard.

**Comment by Chloebarbers** [comment]
hello @2xJDubs 
Yes, it is possible to change collators without unstaking, but it depends on the specific blockchain network and its staking mechanisms.

**Comment by 2xJDubs** [comment]
@Chloebarbers Not sure who you are or why you are even responding here.  Not sure what your scam angle is.

**Comment by boundless-forest** [comment]
Perhaps it's possible if we adjust the contract code. However, whether it's worth doing so is a valid question.
